sycl/source/detail/program_manager/program_manager.hpp

```diff
@@ -346,7 +347,7 @@ class ProgramManager {
   /// Keeps images without entry info.
   /// Such images are assumed to contain all kernel associated with the module.
   /// Access must be guarded by the \ref m_KernelIDsMutex mutex.
-  std::unordered_set<RTDeviceBinaryImage *> m_UniversalKernelSet;
+  std::vector<RTDeviceBinaryImage *> m_UniversalKernelSet;
```

Comment:
Why do we want to use `vector` instead of `unordered_set`?

Reply:
to align with previously existing universal kernel set usage and fix some regressions caused by the change. Removal of universal kernel set will be done later and this will be not needed at all. 
This commit is urgent.

Working on win failures.

Reply:
universal kernel set removal is on its way now https://github.com/intel/llvm/pull/11001